test/libmpv_encode.c

```diff
@@ -115,9 +113,17 @@ int main(int argc, char *argv[])
         return 1;
 
     static char path[] = "./testout.XXXXXX";
+
+#ifdef _WIN32
     out_path = mktemp(path);
     if (!out_path || !*out_path)
         fail("tmpfile failed\n");
+#else
+    int ret = mkstemp(path);
```

Comment:
```suggestion
    int fd = mkstemp(path);
```
just for readability